api_app/services/aad_authentication.py

```diff
@@ -1,60 +1,75 @@
 import base64
 import logging
+from typing import List
 import jwt
 import requests
 import rsa
 
 from fastapi import Request, HTTPException, status
-from fastapi.security import OAuth2AuthorizationCodeBearer
+from msal import ConfidentialClientApplication
```

Comment:
why needed?

Reply:
Line 153. For connecting to Graph. I moved all the AzureAD stuff into this class.